packages/trace-viewer/src/ui/workbench.tsx

```diff
@@ -176,7 +176,7 @@ export const Workbench: React.FunctionComponent<{
   const callTab: TabbedPaneTabModel = {
     id: 'call',
     title: 'Call',
-    render: () => <CallTab action={activeAction} sdkLanguage={sdkLanguage} />
+    render: () => <CallTab action={activeAction} executionStartTime={model?.startTime ?? 0} executionStartWallTime={model?.wallTime ?? Date.now()} sdkLanguage={sdkLanguage} />
```

Comment:
It should never happen that `model?.wallTime` is undefined when `activeAction` is not null, why pass `Date.now()` rather than 0 or -1?  Would be cleaner if `CallTab` accepted single parameter `{ action, startTime, startWallTime } | undefined`, can we do that?

Reply:
Doing so requires changing the component significantly so that we properly represent `model` and `activeAction` as being intertwined state. Currently TypeScript doesn't know they're related.

I would prefer to not change this and leave the coalescence placeholders, but I will change this if you so desire.

Reply:
Doing so requires changing the component significantly so that we properly represent model and activeAction as being intertwined state. Currently TypeScript doesn't know they're related.

I would prefer to not change this and leave the coalescence placeholders, but I will change this if you so desire. I personally think this would be overly specific type handling, though it's obviously more correct.